src/main/java/zmaster587/advancedRocketry/dimension/DimensionManager.java

```diff
@@ -104,7 +105,7 @@ public DimensionManager() {
 
 		overworldProperties = new DimensionProperties(0);
 		overworldProperties.setAtmosphereDensityDirect(100);
-		overworldProperties.averageTemperature = 100;
+		overworldProperties.averageTemperature = 286;
```

Comment:
Could put a comment explaining why the value is 286?  It seems pretty arbitrary

Reply:
Absolutely, you're right that it needs it.